### PR TITLE
[bn] Localize devops.md

### PR DESCRIPTION
**Pull request by ikramulkayes**
Signed-off-Ikramul Kayes ikramul.kayesgg@gmail.com
Localizes `content/bn/devops.md` to Bengali
This is an initial PR as may need some improvements!
Tracking issue: #410

### CODE REVIEW

File: content/bn/devops.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ category: ধারণা
...
+ঐতিহ্যগতভাবে, জটিল সংস্থা সাথে [tightly-coupled](/tightly_coupled_architectures/) [monolithic apps](/monolithic_apps/) , কাজ সাধারণত একাধিক দলের মধ্যে খণ্ডিত ছিল. এটি অসংখ্য হ্যান্ডঅফ এবং দীর্ঘ পরবর্তী সময়, প্রতিবার একটি উপাদান বা আপডেট প্রস্তুত ছিল, এটি পরবর্তী দলের জন্য একটি সারিতে স্থাপন করা হয়েছিল। যেহেতু ব্যক্তিরা কেবলমাত্র প্রকল্পের একটি ছোট অংশে কাজ করেছিল, এই পদ্ধতির ফলে মালিকানার অভাব দেখা দেয়। তাদের লক্ষ্য ছিল পরবর্তী দল কাছে কাজটি পৌঁছে দেওয়া, গ্রাহকের কাছে সঠিক কার্যকারিতা সরবরাহ না করা - অগ্রাধিকারগুলির একটি স্পষ্ট বিভ্রান্তি।
 
-By the time code finally got into production, it went through so many developers, waiting in so many queues that it was difficult to trace the origin of the problem if the code didn’t work. DevOps turns this approach upside down.
+কোডটি শেষ পর্যন্ত  আসার সময় পর্যন্ত, এটি এত বেশি ডেভেলপারের মধ্য দিয়ে গিয়েছিল, এত সারিতে অপেক্ষা করেছিল যে কোডটি কাজ না করলে সমস্যার উত্স খুঁজে বের করা কঠিন ছিল। ডেভওপস এই পদ্ধতিকে উল্টো করে দেয়।
```
Comment (mitul3737) [inline content/bn/devops.md:13]: Kindly change the spelling `উত্স`

**Review by mitul3737 — CHANGES_REQUESTED** [review]
Kindly change the spelling of `উত্স`

File: content/bn/devops.md (hunk 1 above)
Comment (ikramulkayes) [reply content/bn/devops.md]: Done, check it now.

**Review by mitul3737 — APPROVED** [review]

**Review by Mouly22 — APPROVED** [review]
Nice work! Everything is okay

**Pull request merged**